pages/search.js

```diff
@@ -38,9 +38,11 @@ export async function getStaticProps() {
 export default function Search({ data }) {
   let { users, tags } = data;
   const router = useRouter();
+  let randomMin = Math.floor(Math.random() * (users?.length - 5 - 0 + 1)) + 0;
```

Comment:
Not sure I understand what `- 0` and +`0` does?

Reply:
I think it would be cleaner to shuffle the array and select the first 5

You can shuffle with this...

```
array.sort(() => 0.5 - Math.random())
```